tasks/Tasks/provision_aws_cluster.robot

```diff
@@ -0,0 +1,32 @@
+*** Settings ***
+Documentation    Provision AWS Self-Managed clusters for testing Beta
+Metadata         provision
+Resource         ../Resources/Provisioning/Hive/AWS/oc-provision.robot
+Library          OperatingSystem
+Library          OpenShiftLibrary
+Library          String
+Library          Process
+
+***Variables***
+&{infrastructure_configurations}
+
+*** Tasks ***
+Provision Self-Managed Cluster In AWS &{infrastructure_configurations}
+    [Tags]  aws_self_managed_provision
+    Run AWS Configuration In Hive    &{infrastructure_configurations}
```

Comment:
how this variable is filled in?

Reply:
it is filled with the infrastructure yaml file

Reply:
oh you are using that file as "test-variables.yaml". I missed that in the command, sorry